src/signals/incident-management/containers/Dashboard/styled.ts

```diff
@@ -3,8 +3,15 @@
 import { themeColor } from '@amsterdam/asc-ui'
 import styled from 'styled-components'
 
+import { TabContainer } from '../../../../components/Tab'
+
 export const StyledRow = styled.div`
   width: 100%;
   background-color: ${themeColor('tint', 'level2')};
   margin: 0;
 `
+
+export const StyledTabContainer = styled(TabContainer)`
+  margin-top: 1.5rem;
+  margin-left: auto;
```

Comment:
It currently has 28px margin left, which is big and not according to design. Better to make it 16px as designed. (looks better too ;) )

Reply:
well, actually with Linda, we agreed it should be the same as in overview

Reply:
we could make it 16 in both cases though